### ISSUE TITLE
Are these results before and after hyperparameter optimization acceptable?

### ISSUE THREAD

**Issue by skhonsha** [AI4Finance-Foundation/FinRL#373]
`Hyperparameters after tuning {'buffer_size': 10000, 'learning_rate': 0.005414413211338522, 'batch_size': 64}

Hyperparameters before tuning {'batch_size': 128, 'buffer_size': 50000, 'learning_rate': 0.001}`

`
#Backtesting with our pruned model
print("==============Get Backtest Results===========")
now = datetime.datetime.now().strftime('%Y%m%d-%Hh%M')

perf_stats_all_tuned = backtest_stats(account_value=df_account_value_tuned)
perf_stats_all_tuned = pd.DataFrame(perf_stats_all_tuned)
perf_stats_all_tuned.to_csv("./"+config.RESULTS_DIR+"/perf_stats_all_tuned_"+now+'.csv')
`
==============Get Backtest Results===========
Annual return          3.395301
Cumulative returns     3.661268
Annual volatility      3.939871
Sharpe ratio           1.775754
Calmar ratio           4.126460
Stability              0.363702
Max drawdown          -0.822812
Omega ratio            1.933435
Sortino ratio          4.551413
Skew                        NaN
Kurtosis                    NaN
Tail ratio             1.177493
Daily value at risk   -0.468614
dtype: float64

`
#Backtesting for not tuned hyperparamters
print("==============Get Backtest Results===========")
now = datetime.datetime.now().strftime('%Y%m%d-%Hh%M')

perf_stats_all = backtest_stats(account_value=df_account_value)
perf_stats_all = pd.DataFrame(perf_stats_all)
`

==============Get Backtest Results===========
Annual return          4.588587
Cumulative returns     4.983521
Annual volatility      2.427726
Sharpe ratio           1.783085
Calmar ratio           7.818184
Stability              0.569167
Max drawdown          -0.586912
Omega ratio            1.570338
Sortino ratio          3.562616
Skew                        NaN
Kurtosis                    NaN
Tail ratio             1.764417
Daily value at risk   -0.288687
dtype: float64

**Comment by Athe-kunal** [comment]
You have a validation set between your training and testing set?

**Comment by skhonsha** [comment]
Dear @Athe-kunal, Can I have your email to send you the notebook?

**Comment by Athe-kunal** [comment]
Yeah sure
athekunal@gmail.com
